File: docs/APIs-and-SDKs/SDK-Documentation/getting-started/create-new-context-request/rust/async.rs
use absmartly_sdk::ContextOptions;

let units = [("session_id", "5ebf06d8cb5d8137290c4abb64155584fbdb64d8")];

let context_data: ContextData = serde_json::from_str(&api_response)?;

let options = ContextOptions {
    publish_delay: 100,
    refresh_period: 0,
    event_logger: None,
};

let mut context = sdk.create_context_with(units, context_data, Some(options));
assert!(context.is_ready());

File: docs/APIs-and-SDKs/SDK-Documentation/getting-started/create-new-context-request/scala/async.scala
import scala.concurrent.Await
import scala.concurrent.duration._

val units = Map(
  "session_id" -> "5ebf06d8cb5d8137290c4abb64155584fbdb64d8",
  "user_id" -> "123456"
)

val contextFuture = sdk.createContext(units)

val context = Await.result(contextFuture, 5.seconds)
